modular_citadel/code/modules/client/preferences_savefile.dm

```diff
@@ -35,4 +35,5 @@
 	WRITE_FILE(S["feature_xeno_head"], features["xenohead"])
 	//flavor text
 	WRITE_FILE(S["feature_flavor_text"], features["flavor_text"])
+	WRITE_FILE(S["silicon_feature_flavor_text"], features["silicon_flavor_text"])
```

Comment:
why is this here? put it where i put ooc nnotes sving/loading for characters
actually
@Ghommie can you take a look you touched savefiles last i'm a bit confused on what is going on regarding flavor_text vs feature_flavor_text

Reply:
`feature_flavor_text` is the new var being used as a save key while `flavor_text` is the old one

Reply:
@kevinz000 honestly i had no other clue wherelse to put it, so i slapped it in the same place other flavortext was handled.

Reply:
old flavor text can probably be removed from the lists since I doubt anyone with characters from 2016 still exist

Also it was made a DNA feature as a lazy/easy fix for Changlings to not get metagame outed by having the wrong flavor text.